src/com/naren/lists/List_Collections.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.naren.lists;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ *
+ * @author Narender Reddy
+ */
+public class List_Collections {
```

Comment:
Please use camel case for class name and avoid _ while doing it in java. 
https://google.github.io/styleguide/javaguide.html#s5-naming